geomstats/geometry/connection.py

```diff
@@ -627,22 +627,29 @@ def path(t):
 
         return path
 
-    def parallel_transport(self, tangent_vec_a, tangent_vec_b, base_point):
+    def parallel_transport(
+        self, tangent_vec_a, base_point, tangent_vec_b=None, end_point=None
+    ):
         r"""Compute the parallel transport of a tangent vector.
 
         Closed-form solution for the parallel transport of a tangent vector a
-        along the geodesic defined by :math:`t \mapsto exp_(base_point)(t*
-        tangent_vec_b)`.
+        along the geodesic between two points `base_point` and `end_point`
+        or alternatively defined by :math:`t\mapsto exp_(base_point)(
+        t*tangent_vec_b)`.
 
         Parameters
         ----------
         tangent_vec_a : array-like, shape=[..., {dim, [n, m]}]
             Tangent vector at base point to be transported.
+        base_point : array-like, shape=[..., {dim, [n, m]}]
```

Comment:
how about `initial_point` to be consistent with the geodesic's API?

Reply:
Ok